src/platform/site-wide/user-nav/tests/e2e/00-required.cypress.spec.js

```diff
@@ -135,7 +135,7 @@ describe('Site-wide Search functionality with typeahead enabled', () => {
       .children()
       .should('have.length', 5);
     cy.get('[data-e2e-id="sitewide-search-submit-button"]').focus();
-    cy.get('#suggestions-list').should('not.be.visible');
+    cy.get('#suggestions-list').should('not.exist');
```

Comment:
I haven't dug into the commit history, but this must be a new test.  Hooray for swapping out a [deprecated approach for the new one](https://docs.cypress.io/guides/references/migration-guide.html#Non-existent-element-assertions).

Reply:
I ran the tests locally twice in headless mode (`yarn cy:run`), and both times the following two tests failed:

- `src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js`
- `src/applications/personalization/view-representative/tests/view-representative.cypress.spec.js`

I'll try running the tests in Cypress ^4.8.0 to compare...

Reply:
Those same two tests failed in Cypress 4.8.0 too so the issue with them doesn't seem to be Cypress 6.2.1.

You stated you ran the tests in Jenkins and CircleCI and they passed so I'm okay to approve.